recipe_linter.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Since these are specific to github and I like clean root directories, perhaps these can be moved to the `.github` folder?